From 728ca05e835bf4719ab7a8232c9f757c10868c24 Mon Sep 17 00:00:00 2001
From: Christian Bruel <christian.bruel@foss.st.com>
Date: Mon, 12 Jan 2026 15:07:38 +0100
Subject: [PATCH] arm64: configs: fix PCIE PTM configuration in addons fragment

Rename CONFIG_PTM to CONFIG_PCIE_PTM

Signed-off-by: Christian Bruel <christian.bruel@foss.st.com>
Change-Id: I1750b7d19ddd09c1288122f9d87e19abec00d2e6
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/523759
Reviewed-by: Amelie DELAUNAY <amelie.delaunay@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Domain-Review: Amelie DELAUNAY <amelie.delaunay@foss.st.com>
---
 arch/arm64/configs/fragment-02-defconfig-addons.config | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/arch/arm64/configs/fragment-02-defconfig-addons.config
+++ b/arch/arm64/configs/fragment-02-defconfig-addons.config
@@ -55,13 +55,13 @@ CONFIG_NVMEM_STM32_ROMEM=y
 CONFIG_NVMEM_STM32_TAMP=y
 CONFIG_OF_OVERLAY=y
 CONFIG_PCIEASPM_POWERSAVE=y
+CONFIG_PCIE_PTM=y
 CONFIG_PCIE_STM32=m
 CONFIG_PCIE_STM32_EP=m
 CONFIG_PCI_ENDPOINT_MSI_DOORBELL=y
 CONFIG_PHY_STM32_COMBOPHY=m
 CONFIG_PHY_STM32_USB2PHY=m
 CONFIG_PINCTRL_STM32_HDP=m
-CONFIG_PTM=y
 CONFIG_PWM_STM32=m
 CONFIG_PWM_STM32_LP=m
 CONFIG_REGULATOR_ARM_SCMI=y
